src/library_fs.js

```diff
@@ -1421,6 +1421,7 @@ FS.staticInit();` +
     ensureErrnoError: () => {
       if (FS.ErrnoError) return;
       FS.ErrnoError = /** @this{Object} */ function ErrnoError(errno, node) {
+        this.name = 'ErrnoError';
```

Comment:
Please add a comment here explaining why we prefer to use the name to identify instances.

Reply:
Added.